WcaOnRails/app/views/teams/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<% provide(:title, 'Teams') %>
+
+<div class="container">
+  <h1><%= yield(:title) %></h1>
+
+  <% can_edit = current_user.can_edit_users? %>
+  <% if can_edit %>
+    <%= link_to "New team", new_team_path %>
+  <% end %>
+
+  <% if @teams.empty? %>
+    <div class="alert alert-warning">There are currently no teams.</div>
+  <% else %>
+    <table class="table">
+      <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Description</th>
+        <th>Members</th>
+        <th>Leader(s)</th>
+        <th></th>
+      </tr>
+      <% @teams.each do |team| %>
+        <tr>
+          <td><%= team.friendly_id %></td>
+          <td><%= team.name %></td>
+          <td><%= team.description %></td>
+          <td><%= team.team_members.map { |member| member.user.name }.to_sentence %></td>
+          <td>
+            <% team.team_members.each do |member| %>
```

Comment:
Hmm... I feel like this would be cleaner as something like `<%= team.team_members.where(team_leader: true).map { |tm| tm.user.name }.sort.to_sentence %>`.